.teamcity.properties

```diff
@@ -1,6 +1,6 @@
 # Collection of properties customizing TeamCity build configurations
 ReleaseLabel = rtm
-ReleaseProductVersion = 4.6.0
+ReleaseProductVersion = 4.6.1
```

Comment:
Can you delete this file instead of updating it for no reason every time?

Reply:
It's the 4.6.1 branch. 

Don't really want to do it here. 

If you think it needs done, let's do it in dev, not here.

Reply:
Then don’t make this change either. It’s useless and unnecessary

Reply:
Then don’t make this change either. It’s useless and unnecessary

Reply:
I really just looked at an old bump of the version, and just did the same logic. 
It's a temp branch that's only going to be used for these hotfixes. I'm not too concerned with how clean it is :)

Reply:
which is why cleaner it is , the better :P